scripts/contiv-error-report.sh

```diff
@@ -76,35 +74,32 @@ do
     NODE_NAMES+=("${NODE_FIELDS[0]}")
 done
 
-# Print header
-NODE_NAME_LEN=$(max_string_length "${NODE_NAMES[@]}")
-
 for nn in "${NODE_NAMES[@]}"
 do
     # Print node header
-    printf "%s:\n" "$nn"
+    printf "%s:\\n" "$nn"
     printf '%0.s=' $( seq 1 $(( ${#nn} + 1 )) )
     echo
 
-    VSWITCH_LOG_FILE_NAME=$( ls "$nn" | grep "$VSWITCH_LOG" )
+    VSWITCH_LOG_FILE_NAME=$( get_log_file_name "$nn" "$VSWITCH_LOG" )
```

Comment:
This seems to be doing a grep without using a delimiter, and possibly VSWITCH_LOG may contain special regex characters that might need to be escaped.